tests/e2e/specs/in-game/spectating.spec.js

```diff
@@ -358,7 +358,7 @@ describe('Spectating Games', () => {
         .should('not.contain', 'Request Stalemate')
         .should('not.contain', 'Concede');
       cy.get('[data-cy="stop-spectating"]').click();
-      cy.hash().should('eq', '#/');
+      cy.location('pathname').should('eq', '/');
```

Comment:
We no longer can use the hash, we need to use the actual pathname of the route in these tests.